app/export.go

```diff
@@ -76,131 +75,15 @@ func (app *ManifestApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAdd
 
 	/* Handle fee distribution state. */
 
-	// withdraw all validator commission
-	err := app.StakingKeeper.IterateValidators(ctx, func(_ int64, val stakingtypes.ValidatorI) (stop bool) {
```

Comment:
removing this logic scares me. starting a network, exporting, forking the import, and restart with same vals would need to be tested for this. Not sure of any e2e which has this

Reply:
Reverted in [0ceec03](https://github.com/liftedinit/manifest-ledger/pull/76/commits/0ceec034539342d9b25fec8b97f79da8e02c1336)